app/routes/my-sessions/list.js

```diff
@@ -15,7 +15,7 @@ export default Route.extend(AuthenticatedRouteMixin, {
     this.set('params', params);
     let filterOptions = [];
     if (params.session_status === 'upcoming') {
-      filterOptions = [
+      filterOptions.push(
```

Comment:
Dont refactor them. Just add the filter parameters

Reply:
Is this wrong?